.github/workflows/build_validation_workflow.yml

```diff
@@ -2,18 +2,76 @@ name: Build Validation Workflow
 
 on:
   workflow_call:
+    inputs:
+      image_name:
+        type: string
+        description: "Execution Environment"
+        required: false
+      model_type:
+        type: string
+        description: "type of model to execute"
+        required: false
+      IS_DOCKER:
```

Comment:
is there a reason this is ALL_CAPS when compared to the other input params?

Reply:
THERE'S NO REASON FOR IT TO BE IN ALL CAPS 😂